payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SectionUI.kt

```diff
@@ -109,15 +78,18 @@ internal fun SectionTitle(@StringRes titleText: Int?) {
 @Composable
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 fun SectionCard(
+    modifier: Modifier = Modifier,
+    isSelected: Boolean = false,
     content: @Composable () -> Unit
 ) {
     Card(
-        border = BorderStroke(CardStyle.cardBorderWidth, CardStyle.cardBorderColor),
-        elevation = CardStyle.cardElevation,
+        border = PaymentsTheme.getBorderStroke(isSelected),
+        // todo skyler: this will change when we add shadow configurations.
```

Comment:
FYI `// TODO(skyler-stripe): ...` is the format that the IDE recognizes as a TODO.